partialWaveFit/fitResult.cc

```diff
@@ -281,6 +348,12 @@ complex<double>
 fitResult::spinDensityMatrixElem(const unsigned int waveIndexA,
                                  const unsigned int waveIndexB) const
 {
+	// spin density matrix element is 0 if the two waves have different
+	// reflectivities
+	if (partialWaveFitHelper::getReflectivity(_waveNames[waveIndexA])
+	    != partialWaveFitHelper::getReflectivity(_waveNames[waveIndexB]))
+		return 0;
```

Comment:
Could you please add { } here? It is confusing, as the if clause already spans two lines...